packages/itwin/property-grid/src/components/MultiElementPropertyGrid.scss

```diff
@@ -22,10 +22,18 @@
 }
 
 .property-grid-react-animated-tab {
```

Comment:
Is there a need to include "react" in the class name?
property-grid-animated-tab

Reply:
I think its because the package is called "property-grid-react" so they may just be following that convention but I'll wait for others to comment!

Reply:
yea its because the pkg name is property-grid-*react*, and i think we wanted to make sure we didnt have too generic classnames that cuase collisions in other apps. Myabe we need a separate pr to review all the css?